server/service/client_hosts.go

```diff
@@ -99,10 +103,14 @@ func (c *Client) TransferHosts(hosts []string, label string, status, searchQuery
 		return err
 	}
 
+	var teamIDPtr *uint
+	if teamID != 0 {
```

Comment:
So what happens if teamID is 0?

Reply:
`teamIDPtr` stays `nil`, and the host will be assigned to 'No teams'